widget/menu_item.go

```diff
@@ -37,6 +41,25 @@ var keySymbols = map[fyne.KeyName]rune{
 	fyne.KeyUp:        '↑',
 }
 
+// key default text
```

Comment:
This comment does not add any value over the variable name. Either remove it or write something more helpful.

Reply:
Thank you for taking the time to review my PR. 

I will rework the comments.